rune/libenclave/intelsgx/arch.go

```diff
@@ -159,17 +161,30 @@ const (
 )
 
 const (
-	QuoteVersion = 2
+	QuoteVersion2 = 2
+	QuoteVersion3 = 3
 )
 
-type QuoteBody struct {
-	Version       uint16   `struct:"uint16"`
-	SignatureType uint16   `struct:"uint16"`
-	Gid           uint32   `struct:"uint32"`
-	ISVSvnQe      uint16   `struct:"uint16"`
-	ISVSvnPce     uint16   `struct:"uint16"`
-	_             [4]byte  `struct:"[4]byte"`
-	Basename      [32]byte `struct:"[32]byte"`
+type QuoteHeader struct {
+	Version            uint16 `struct:"uint16"`
+	AttestationKeyType uint16 `struct:"uint16"`
```

Comment:
‘AttestationKeyType‘ is used by quote3_header (uint16_t att_key_type; ///< 2: sgx_attestation_algorithm_id_t. ), could you set this section name more neutral name?

Reply:
> ‘AttestationKeyType‘ is used by quote3_header (uint16_t att_key_type; ///< 2: sgx_attestation_algorithm_id_t. ), could you set this section name more neutral name?

LGTM，thanks, I will change the `AttestationKeyType` to `SignatureType`.